api/swagger-spec/oapi-v1.json

```diff
@@ -18166,7 +18166,8 @@
     "required": [
      "strategy",
      "triggers",
-     "replicas"
+     "replicas",
+     "transient"
```

Comment:
Agreed on the name, *transient* is weak. From the rest of the names you have proposed I don't feel strongly on any but eventually would choose *test*. Or maybe *mock*. *temporary* doesn't convey the appropriate meaning. @ironcladlou any opinions here?